src/gcodePlanner.cpp

```diff
@@ -238,7 +238,7 @@ void GCodePlanner::moveInsideCombBoundary(int distance)
     }
 }
 
-GCodePath& GCodePlanner::addTravel(Point p)
+GCodePath& GCodePlanner::addTravel(Point p, bool going_to_outer_wall)
```

Comment:
Or I mean above that, so that `addPolygonsByOptimizer` also gets a parameter `force_retraction`.